diagnostic.rb

```diff
@@ -2,15 +2,42 @@
 # Diagnostic answers to:
 
 # question 2:
+Licensee.create surname: 'Bishop',
+            given_name: 'Caren',
+            gender: 'f',
+            height: 63,
+            weight: 132,
+            born_on: '1943-09-26',
+            eye_color: 'Brown',
+            hair_color: 'Black'
 
 # question 3:
 Licensee.transaction do
   CSV.foreach 'data/licensees.csv', headers: true do |row|
+    person = row.to_hash
+    Person.create! person unless Person.exists? person
   end
 end
 
 # question 4:
+# Use ActiveRecord to retrieve all the licensees with Hazel eye color and
+# Brown or Black hair color.
+Licensee.where( { eye_color: "Hazel", hair_color: ["Brown", "Black"] } )
 
 # question 5:
+# Use ActiveRecord to make each of Marilynn Escobar and Chris Whaley
+# one inch shorter. Look them up by their names.
+marilynn = Licensee.where( { given_name: "Marilynn", surname: "Escobar" } )
+marilynn.update(height -= 1)
+
+chris = Licensee.where( { given_name: "Chris", surname: "Whaley" } )
+chris.update(height -= 1)
```

Comment:
This will be even simpler in Rails 5, coming out later this year. ActiveRecord will have a new `.or` method that can be used as follows:
```ruby
licensees_to_shorten = Licensee.where( 
  given_name: 'Marilynn', surname: 'Escobar'
).or(Licensee.where( given_name: 'Chris', surname: 'Whaley' ))
```